api/EffectTiming.json

```diff
@@ -23,9 +37,11 @@
             "version_added": null
           },
           "opera": {
+            "alternative_name": "AnimationEffectTimingProperties",
             "version_added": true
           },
           "opera_android": {
+            "alternative_name": "AnimationEffectTimingProperties",
             "version_added": true
           },
```

Comment:
Shouldn't this match Chrome's data?

Reply:
I have no way to confirm that.

Reply:
If you can't confirm data for this feature + browser, then why are you changing it? Do you mean to say you tested it manually and this is what you found?

Reply:
Fixed.